Sources/CALayer.swift

```diff
@@ -241,16 +241,23 @@ open class CALayer {
     internal var _presentation: CALayer?
     open func presentation() -> CALayer? { return _presentation }
 
-    var disableAnimations = false
+    internal var disableAnimations = false
 
-    var animations = [String: CABasicAnimation]() {
+    internal var animations = [String: CABasicAnimation]() {
```

Comment:
isn't internal already the default access level?

Reply:
It is, but given that we're making a library for public consumption I'd like to spell out `internal` vars/`func`s wherever they occur. Part of that decision is to make people think twice about making these `public`: if someone has written `internal` it makes it clear that they didn't just forget to write `public` – that it's by design. Does that make sense @michaelknoch ?